examples/adot/docker-compose.yml

```diff
@@ -27,16 +27,12 @@ services:
     networks:
       - my_network
   data-prepper:
+    restart: unless-stopped
     container_name: data-prepper
-    build:
-      context: ../..
-      dockerfile: examples/Dockerfile
-    working_dir: /app
-    command: sh data-prepper-wait-for-odfe-and-start.sh
+    image: opendistroforelasticsearch/data-prepper:v0.7.0-alpha
```

Comment:
this is staging repository - you can as well use production after Dec14

Reply:
Okay. Will follow up with another PR on 14th.